back/app/controllers/web_api/v1/mentions_controller.rb

```diff
@@ -37,4 +51,26 @@ def post_type_to_class(type)
     else raise "#{type} is not a post type"
     end
   end
+
+  # @param [String] query
+  # @param [Idea, Initiative, nil] post
+  # @return [ActiveRecord::Relation]
+  def find_users_by_query(query, post)
+    case params[:roles]
+    when %w[admin moderator]
```

Comment:
Good question. I'll check.

Reply:
You're right. It would not work in reverse order. Good catch. Kinda embarrassed I missed this!

Fixed now, and I also switched the order of the query param array elements in one of the tests, as a simple regression test.